src/zfs/open3.rs

```diff
@@ -267,10 +267,10 @@ pub(crate) fn parse_filesystem_lines(lines: &mut Lines, name: PathBuf) -> Proper
                 properties.exec(parse_bool(&value));
             },
             "filesystem_count" => {
-                properties.filesystem_count(value.parse().expect(FAILED_TO_PARSE));
+                properties.filesystem_count(parse_opt_num(&value));
```

Comment:
i.e. here we can `.unwrap_or_default()` That would make usage of `FilesystemProperties ` easier